dev/Deployment/DeploymentManager.cpp

```diff
@@ -206,21 +219,29 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
 
     // Adds the package at the path using PackageManager.
     // This requires the 'packageManagement' or 'runFullTrust' capabilities.
-    HRESULT DeploymentManager::AddPackage(const std::filesystem::path& packagePath) try
+    HRESULT DeploymentManager::AddPackage(const std::filesystem::path& packagePath, bool forceDeployment) try
     {
-        auto packagePathUri = winrt::Windows::Foundation::Uri(packagePath.c_str());
+        const auto packagePathUri = winrt::Windows::Foundation::Uri(packagePath.c_str());
         winrt::Windows::Management::Deployment::PackageManager packageManager;
-        auto options{ winrt::Windows::Management::Deployment::DeploymentOptions::None };
-        auto deploymentResult{ packageManager.AddPackageAsync(packagePathUri, nullptr, options).get() };
+
+        auto GetDeploymentOptions = [](bool forceDeployment)
+        {
+            return (forceDeployment == true ?
```

Comment:
Nit: ForceDeployment ? winrt::Windows::Management::Deployment::DeploymentOptions::ForceTargetApplicationShutdown:winrt::Windows::Management::Deployment::DeploymentOptions::None);

Reply:
Never explicitly check for == or != for `true` or `false`.